rocketpy/Rocket.py

```diff
@@ -245,7 +239,9 @@ def __init__(
             "linear",
             "constant",
         )
-        self.cpPosition = 0  # Set by self.evaluateStaticMargin()
+        self.centerOfPressure = Function(
+            lambda mach: 0
+        )  # This will be set by the self.evaluateCenterOfPressure()
```

Comment:
The side comment length is causing the linter to divide this simple line into multiple.

Just a aesthetical change, an equivalent approach would be using `Function(0)`, I believe it makes it clearer that is only a temporary value.